tests/task_t_1.spec.js

```diff
@@ -48,3 +44,49 @@ test('T4: Check that URL shortening feature works', async ({ page }) => {
   const newPageURL = await page.url();
   expect(newPageURL).toBe("https://www.wikipedia.org/");
 });
+
+test('T5: Check that intitle functionality works', async ({ page }) => {
+  const searchInputField = page.locator('#search_form_input_homepage');
+  await searchInputField.fill('intitle:panda');
+  await page.click('#search_button_homepage');
+
+  const results = await page.evaluate(() => Array.from(document.querySelectorAll('div [data-nir="1"]'), element => element.textContent));
+        results.forEach(result => {
+            expect(result.toLowerCase()).toContain("panda");
+        });
+});
+
+test('T6: Check that automatic navigation to first result works', async ({ page }) => {
+  const searchInputField = page.locator('#search_form_input_homepage');
+  await searchInputField.fill('!w Lithuania');
+  await page.click('#search_button_homepage');
+  const newLoadedPage = await page.url();
```

Comment:
It would be good to assert the specific wikipedia URL where we were navigated to, e.g. `expect(redirectedUrl).toBe("https://en.wikipedia.org/wiki/Lithuania"); `

Reply:
fixed